docs/_docs/monitoring-metrics/new-metrics.adoc

```diff
@@ -286,23 +286,28 @@ Register name: `io.communication`
 |SentBytesCount | long  |  Sent bytes count.
 |ReceivedBytesCount|  long|    Received bytes count.
 |ReceivedMessagesCount|   integer| Received messages count.
-|RejectedSslSessionsCount|   integer|   TCP sessions count that were rejected due to the SSL errors (metric is exported only if SSL is enabled).
+|RejectedSslSessions|   integer|   TCP sessions count that were rejected due to the SSL errors (metric is exported only if SSL is enabled).
```

Comment:
Here and below. The metric name doesn't match GridNioSslFilter#SSL_REJECTED_SESSIONS_CNT_METRIC_NAME value. RejectedSslSessions vs RejectedSslSessionsCount

Reply:
Fixed.